RoboJackets/urc-software
# [ros2 control] exposed status light interface, allowed for firmware to parse both drivetrain and status light commands

# Description
This PR does the following:
- Fixes the status light interface to send a command to the right port and address
- Allows for both rover drivetrain and status light commands to both be sent to teensy

# Testing Steps (if relevant)
1. colcon build
2. . install/setup.bash
3. ros2 launch urc_bringup bringup.launch.py

Expectation: Should start all hardware interfaces and allow you to drive the drivetrain

# Self Checklist
